src/css_to_emotion.re

```diff
@@ -1126,33 +1125,98 @@ and render_declaration_list = ((list, loc): Declaration_list.t, variables): expr
 }
 and render_style_rule = (ident, sr: Style_rule.t): expression => {
   let (prelude, prelude_loc) = sr.Style_rule.prelude;
-  let selector =
-    List.fold_left(
-      (s, (value, value_loc)) =>
-        switch (value) {
-        | Delim(":") => ":" ++ s
-        | Ident(v)
-        | Operator(v)
-        | Delim(v)
-        | Selector(v) =>
-          if (String.length(s) > 0) {
-            v ++ " " ++ s;
-          } else {
-            v ++ s;
-          }
-        | _ => grammar_error(value_loc, "Unexpected selector")
-        },
-      "",
-      List.rev(prelude),
-    );
-  let selector_expr = string_to_const(~loc=prelude_loc, selector);
   let dl_expr = render_declaration_list(sr.Style_rule.block, None);
-
-  Exp.apply(
-    ~loc=sr.Style_rule.loc,
-    ident,
-    [(Nolabel, selector_expr), (Nolabel, dl_expr)],
-  );
+  switch (prelude) {
+  | [(Selector("&"),_),(Delim(":"),_),(Delim(":"),_),(Ident(pc),loc)] =>
+    let f = switch (pc) {
+    | "active" => "active"
+    | "after" => "after"
+    | "before" => "before"
+    | "first-line" => "firstLine"
+    | "first-letter" => "firstLetter"
+    | "selection" => "selection"
+    | "placeholder" => "placeholder"
+    | _ => grammar_error(loc, "Unexpected pseudo-class")
+    };
+    let ident = Exp.ident(~loc, {txt: Emotion.lident(f), loc});
+    Exp.apply(
+      ~loc=sr.Style_rule.loc,
+      ident,
+      [(Nolabel, dl_expr)],
+    );
+  | [(Selector("&"),_),(Delim(":"),_),(Ident(pc),loc)] =>
```

Comment:
What do you think about adding a comment here explaining the difference between this case and the above one?